### PR TITLE
add test for a untested configuration

### PR DESCRIPTION
**Pull request by leej3**
Test builds that set a NIFTI_PACKAGE_PREFIX and set NIFTI_BUILD_APPLICATIONS

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -68,6 +68,24 @@ matrix:
...
+        - BLDPREFIX=with_applications_
+        - PARALLEL_LEVEL=3
+        - AGENT_BUILDDIRECTORY=${TRAVIS_BUILD_DIR}/NIFTIworkspace
+        - BUILD_SOURCESDIRECTORY=${TRAVIS_BUILD_DIR}
+        - SYSTEM_PULLREQUEST_SOURCEBRANCH=${TRAVIS_PULL_REQUEST_BRANCH}
+        - BUILD_SOURCEBRANCHNAME=${TRAVIS_PULL_REQUEST_BRANCH}
+        - BUILD_BUILDID=${TRAVIS_BUILD_ID}
+        - SYSTEM_PULLREQUEST_PULLREQUESTNUMBER=${TRAVIS_PULL_REQUEST}
+        - CTEST_SCRIPT_DIRECTORY=${TRAVIS_BUILD_DIR}/cmake
+       - INTERPROCEDURAL_OPTIMIZATION="ON"
```
Comment (hjmjohnson) [inline .travis.yml:83]: Super minor, and does not need to be fixed, but this is not indented the same as surrounding items.

**Comment by leej3** [comment]
Oh, missed that. Duly noted. It seems I haven't addressed this use case.
It's the last circleci test of gifti_clib. I haven't worked out why it has
only failed now but it looks like I should fix/test the specific
configuration here.

File: nifti2/CMakeLists.txt [hunk 2]
```diff
@@ -15,9 +15,10 @@ endif()
 install_nifti_target(${NIFTI_NIFTILIB2_NAME})
 
 if(NIFTI_BUILD_APPLICATIONS)
-  add_nifti_executable(nifti_tool nifti_tool.c)
-  target_link_libraries(nifti_tool PUBLIC ${NIFTI_NIFTILIB2_NAME})
-  install_nifti_target(nifti_tool)
+  set(NIFTI_TOOL ${NIFTI_PACKAGE_PREFIX}nifti_tool)
```
Comment (hjmjohnson) [inline nifti2/CMakeLists.txt:18]: This looks correct to me.

Comment (leej3) [reply nifti2/CMakeLists.txt:18]: Yep. I think that snook in with my changes. Apologies. I had been attempting to migrate to less variables. Its tested now as is.

**Pull request merged**